src/index.ts

```diff
@@ -138,3 +139,54 @@ export function useLiveAnchorAccount<
     slotUpdated,
   };
 }
+
+function fetcher<I extends Idl, A extends keyof IdlAccounts<I>>(
+  program: Program<I>,
+) {
+  return function (method: A, ...params: [Address]): IdlAccounts<I>[A] {
+    const address = params[0];
+    return program.account[method].fetch(address) as IdlAccounts<I>[A];
```

Comment:
```suggestion
  return function (method: A, ...params: [Address]): Promise<IdlAccounts<I>[A]> {
    const address = params[0];
    return program.account[method].fetch(address) as Promise<IdlAccounts<I>[A]>;
```

on a side-note i have no idea why i cant get the type-checking to pass without the use of `as`,  on `line 103`